homepage/de.js

```diff
@@ -0,0 +1,67 @@
+{
+	"Integrations": "Integrationen",
+	"Pricing": "Preise",
+	"Apps": "Apps",
+	"Existing user login": "Als bestehender Nutzer anmelden",
+	"login": "anmelden",
+	"Chat, for GitHub.": "Chat, f&uuml;r GitHub.",
+	"Get unlimited public rooms and one-to-one chats, free forever. Private team plans start from $2 per user per month. No credit card required.": "Erhalte unbegrenzte &ouml;ffentliche und private Chatrooms. Bereits ab $2 im Monat pro Benutzer. Keine Kreditkarte notwendig.",
```

Comment:
This translation is inaccurate – „private Chatrooms“ is vague, and the second sentence implies that *any* service costs $2 per month.

I would suggest the following instead:

> Erhalte unbegrenzte, *dauerhaft kostenfreie* öffentliche Chatrooms und *Einzelgespräche. Nichtöffentliche Chatrooms gibt es bereits* ab $2 im Monat pro Benutzer. Keine Kreditkarte notwendig.

And perhaps I’d drop the „Erhalte“, since it feels a bit more awkward than “Get” in English:

> Unbegrenzte, dauerhaft kostenfreie öffentliche Chatrooms und Einzelgespräche. …